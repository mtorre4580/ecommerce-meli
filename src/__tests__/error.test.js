import React from 'react';
import Error from '../components/commons/error/error';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

describe('<Error />', () => {

    let title = 'Se produjo un error';
    let content = 'No se pudieron obtener resultados';

    it('should render snapshot', () => {
        const component = renderer.create(<Error title={title} content={content} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should be equal title and condition props', () => {
        const wrapper = mount(<Error title={title} content={content} />);
        expect(wrapper.props().title).toEqual(title);
        expect(wrapper.props().content).toEqual(content);
    });
     
 });


