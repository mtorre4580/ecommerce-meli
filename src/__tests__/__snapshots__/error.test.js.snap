// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should render snapshot 1`] = `
<div
  className="error-ctn"
>
  <h2
    className="title-err"
  >
    Se produjo un error
  </h2>
  <p>
    No se pudieron obtener resultados
  </p>
</div>
`;
